pom.xml

```diff
@@ -1354,7 +1354,8 @@
         <artifactId>maven-surefire-plugin</artifactId>
         <version>2.22.2</version>
         <configuration>
-          <argLine>-Dfile.encoding=UTF-8</argLine>
+          <!-- Minimal stack size for early detection of stack overflow issues. -->
+          <argLine>-Xss256K -Dfile.encoding=UTF-8</argLine>
```

Comment:
CI failure was during regression run. Are you sure this should be in checkstyle's POM and not in regression's POM?

Regression doesn't use checkstyle's POM, it just uses the checkstyle built artifact. I wouldn't think it transfers.